examples/notebooks/beam-ml/mltransform/compute_and_apply_vocab.ipynb

```diff
@@ -0,0 +1,326 @@
+        "artifact_location_with_frequency_threshold = tempfile.mkdtemp(prefix='compute_and_apply_vocab_frequency_threshold_')"
+      ],
+      "metadata": {
+        "id": "wkVNu3SJrYnc"
+      },
+      "execution_count": 17,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "documents = [\n",
+        "    {\"feature\": \"the quick brown fox jumps over the lazy dog\"},\n",
+        "    {\"feature\": \"the five boxing wizards jump quickly in the sky\"},\n",
+        "    {\"feature\": \"dogs are running in the park\"},\n",
+        "    {\"feature\": \"the quick brown fox\"}\n",
+        "]"
+      ],
+      "metadata": {
+        "id": "s3bYkvXlrnRt"
+      },
+      "execution_count": 18,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "In this example, `MLTransform` in `write` mode uses `ComputeAndApplyVocabulary` to generate vocabulary on the incoming dataset. The generated vocabulary is stored in an `artifact_location` that you can use on a different dataset in `read` mode with `MLTransform`."
+      ],
+      "metadata": {
+        "id": "oETBJNVfRws_"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "with beam.Pipeline() as pipeline:\n",
+        "  data_pcoll = pipeline | \"CreateData\" >> beam.Create(documents)\n",
+        "  # Compute and apply vocabulary by using MLTransform.\n",
+        "  transformed_pcoll = (\n",
+        "      data_pcoll\n",
+        "      | \"MLTransform\" >> MLTransform(write_artifact_location=artifact_location).with_transform(\n",
+        "          ComputeAndApplyVocabulary(columns=['feature'], split_string_by_delimiter=' '))\n",
+        "      )\n",
+        "  transformed_pcoll | \"Print\" >> beam.Map(print)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "6hAYN0ZYt2da",
+        "outputId": "b11ba556-b124-4d1f-c82a-8b183129e5b5"
+      },
+      "execution_count": 19,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "WARNING:absl:You are passing instance dicts and DatasetMetadata to TFT which will not provide optimal performance. Consider following the TFT guide to upgrade to the TFXIO format (Apache Arrow RecordBatch).\n",
+            "WARNING:absl:You are passing instance dicts and DatasetMetadata to TFT which will not provide optimal performance. Consider following the TFT guide to upgrade to the TFXIO format (Apache Arrow RecordBatch).\n",
+            "WARNING:absl:You are outputting instance dicts from `TransformDataset` which will not provide optimal performance. Consider setting  `output_record_batches=True` to upgrade to the TFXIO format (Apache Arrow RecordBatch). Encoding functionality in this module works with both formats.\n"
```

Comment:
Before merging, we should manually remove these warnings. I don't think they're a big deal in the colab itself, but they're annoying when rendered in GitHub or other docs